contracts/bridge/Bridge.sol

```diff
@@ -172,45 +172,43 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
     }
 
     // _requestKLAYTransfer requests transfer KLAY to _to on relative chain.
-    function _requestKLAYTransfer(address _to, uint256 _fee) internal {
+    function _requestKLAYTransfer(address _to, uint256 _feeLimit) internal {
         require(isRunning, "stopped bridge");
-        require(msg.value > 0, "zero msg.value");
-        require(msg.value > _fee, "insufficient amount");
-        require(feeOfKLAY == _fee, "invalid fee");
+        require(msg.value > _feeLimit, "insufficient amount");
 
-        _payKLAYFee(_fee);
+        uint256 fee = _payKLAYFeeAndRefundChange(_feeLimit);
 
         emit RequestValueTransfer(
             TokenKind.KLAY,
             msg.sender,
-            msg.value.sub(_fee),
+            msg.value.sub(_feeLimit),
             address(0),
             _to,
             requestNonce,
             "",
-            _fee
+            fee
         );
         requestNonce++;
     }
 
     // () requests transfer KLAY to msg.sender address on relative chain.
     function () external payable {
-        _requestKLAYTransfer(msg.sender, feeOfKLAY);
+        _requestKLAYTransfer(msg.sender, msg.value);
```

Comment:
I'm not sure this is right. If we call like this, msg.value == _feeLimit in _requestKLAYTransfer().
In this case, msg.value.sub(_feeLimit) == 0. Is this intentional?

Reply:
In this case, I think designated fee can be input as feeLimit.

Reply:
`_requestKLAYTransfer` requires `msg.value > _feeLimit `. Is it always reverted? It will be nice to add a test case for the payable method.

Reply:
I reverted it and added a test case for empty payable method. :) Thanks.